edb/edgeql/ast.py

```diff
@@ -66,6 +66,12 @@ class CardinalityModifier(s_enum.StrEnum):
     Required = 'REQUIRED'
 
 
+class DescribeGlobal(s_enum.StrEnum):
+    Schema = 'SCHEMA'
+    SystemConfig = 'SYSTEM CONFIG'
+    Roles = 'ROLES'
```

Comment:
Maybe `DESCRIBE ROLE;` would be OK to describe all roles? That way we don't need to add another unreserved keyword (which are still perceived as regular keywords by users).